cmip_basins/gfdl.py

```diff
@@ -144,12 +142,12 @@
         [120.8, 21.9],  # southern tip taiwan
         [121.9, 25.0],  # gongliao taiwan
         [130.0, 30.8],  # kagoshima
-        [130.0, 33.6],  # fukuoka
-        [135.4, 34.7],  # osaka
+        [130.95,32.83],  # kumamoto
```

Comment:
Looks like the formatter is complaining about the lack of a space between these array elements.  Rerunning black should fix it,